app/views/content/landing/how-to-become-a-teacher/_steps.html.erb

```diff
@@ -0,0 +1,35 @@
+<div class="row">
+  <div class="grey col">
+    <section class="col col-720 centered">
+      <p class="inset">
+        We can help you to understand what's involved and decide if a
+        career teaching in a primary or secondary school in England is right for you.
+      </p>
+
+      <ol class="steps">
+        <li class="step">
+          <header class="step__header">
+            <div class="step__number">1</div>
+            <h2 class="heading-m">What qualifications do I need?</h2>
+          </header>
+        </li>
+        <li class="step">
+          <header class="step__header">
+            <div class="step__number">2</div>
+            <h2 class="heading-m">Can I get help funding my training?</h2>
+          </header>
+        </li>
+        <li class="step">
+          <header class="step__header">
+            <div class="step__number">3</div>
+            <h2 class="heading-m">How do I find and apply for teacher training?</h2>
+          </header>
+        </li>
+      </ol>
+
+      <p class="inset">
+        Read on to find out more.
```

Comment:
```suggestion
```

Reply:
Agree that this isn't needed? Need to remove the <p>s as well though?